src/components/v5/shared/Button/Button.module.css

```diff
@@ -26,6 +26,10 @@
   @apply bg-base-white text-blue-400 border border-blue-400 hover:bg-blue-400 hover:border-blue-400 hover:text-base-white disabled:text-gray-300 disabled:border-gray-300;
 }
 
+.senary {
+  @apply w-full flex items-center justify-start py-2 px-4 hover:bg-gray-50 hover:font-medium transition-all duration-normal rounded;
```

Comment:
`flex items-center transition-all duration-normal`
these styles (and maybe some more? check it please) are added directly to the button

Reply:
Removed all doubled styles